### ISSUE TITLE
Holy Resistance

### ISSUE THREAD

**Issue by Zindai** [ccshiro/cc-buglist#1778]
This report has been marked as invalid, and with good reason. I'll be leaving the original report further down in the post.

See Barroth's [link to EJ](http://forums.elitistjerks.com/topic/9411-spell-hit-spell-crit-and-mages/page-2):
_Empirical evidence shows that mobs that are higher level than yourself have an innate chance of partially resisting spell damage that is impossible to counter._

See also Wowwiki's [article on Magical Resistance](http://wowwiki.wikia.com/wiki/Formulas:Magical_resistance?oldid=1603715):
_Level-based resistance (not to be confused with level-based miss) can play a factor in total resists. For every level that a mob has over the player, there is 8 resist (believed; the exact number may be higher) added. For boss fights, this means there is 15-24 resistance added. This extra resistance means there will be partial resists on non-binary spells from the added resistance. However, this resistance has been shown to not apply to binary spells at all. This level based resistance cannot be reduced by any means, not even Spell Penetration._

---

Original report:

**How it should work**
Partial resists **only** exist as a result of resistances, while full resists may be a result of resistances **or** chance to be missed. However, holy resistance does not exist as a stat, and only few mobs have some form of protection against holy spells.

**PvE on Corecraft**
Level 3 priest casting Smite vs level 4 mob: Full hits, **partial resists**, full resists.
Level 4 priest casting Smite vs level 4 mob: Full hits, no partial resists, full resists.

This was the case for 20+ mobs, and the partial resists show that the higher level mob has holy resistance, which it shouldn't have. It's likely built into mobs through "all resistances" or chance to be missed.

**PvP on Corecraft**
Between players there seems to be no issue, even with level difference, Mark of the Wild and "resist all" enchants. A holy spell was never partially resisted (only hit or full resist), but other schools were.

---

**In short**
Higher level mobs should only have an increased chance to **fully** resist holy spells, which is caused by not being capped on spell hit. A mob should not be able to **partially** resist holy spells, which would be caused by having holy resistance.

---

**Discussions on the subject**
http://wowwiki.wikia.com/wiki/Magic_schools_(WoW)#Holy
http://wowwiki.wikia.com/wiki/Resistance?oldid=1561363
http://www.excalibur-nw.com/forum/tracker/issue-2323-holy-damage-being-partially-resisted-mobs-having-holy-resistance/
http://www.mmo-champion.com/threads/676034-Holy-Resistance

---

Last edited: 2015-09-14 22:30 (10:30 PM) CEST

**Comment by Barroth** [comment]
Those partial resists are caused by a mechanic called level based resistance, where mobs do not actually have the specific resistance, but are able to partially(or fully for nonbinary spells) resist the spell. For infos about that mechanic read: https://dwarfpriest.wordpress.com/2008/01/07/spell-hit-spell-penetration-and-resistances/#more-176

Some combatlog values(from 1.12) show smite partial resists aswell: http://forums.elitistjerks.com/topic/9411-spell-hit-spell-crit-and-mages/?p=264535

Thus I think this is working as intended.

**Comment by Zindai** [comment]
I'll have a closer look at your links later. Level based resistance giving partial resists does go against what I've read about partial resists so far, but if it's a matter of "resisting all magic" rather than resisting schools, I can see how you would be correct.

**Comment by ccshiro** [comment]
Reading the thread I realized Barroth posted evidence disproving the claim, as such I will just close as invalid.
